### PR TITLE
feat(frontend/backend/shared): compare button tm-622

### PR DESCRIPTION
**Pull request by Ann-Maksay**
- Button hover:
compare-buttonv3

- Flow
compare-flow

- Unsubscribed 
compare-unsub
compare-unsab
compare-unsab-mobile

- Subscribed
compare-sab
compare-sab-tablet
compare-sab-mobile

### CODE REVIEW

File: apps/frontend/src/libs/components/courses/courses.tsx [hunk 1]
```diff
@@ -11,22 +12,33 @@ import { Course } from "../course/course.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	commonCourses?: number[];
```
Comment (what1s1ove) [inline apps/frontend/src/libs/components/courses/courses.tsx:15]:
```suggestion
	commonCoursesIds?: number[];
```
check and change it everywhere

File: apps/frontend/src/modules/user-courses/slices/user-courses.slice.ts [hunk 2]
```diff
@@ -14,7 +13,7 @@ import {
 
 type State = {
 	addCourseDataStatus: ValueOf<typeof DataStatus>;
-	commonCourses: CourseDto[];
+	commonCourses: number[];
```
Comment (what1s1ove) [inline apps/frontend/src/modules/user-courses/slices/user-courses.slice.ts:16]:
```suggestion
	commonCoursesIds: number[];
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]
do not forget to add this feature to subscription benefits

**Comment by Ann-Maksay** [comment]
> do not forget to add this feature to subscription benefits

image

It's already there

**Review by what1s1ove — APPROVED** [review]
Let's wait for @MappariChan now

**Review by Oleksandra-Masiuk — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**